attestation-agent/kbs_protocol/src/bin/trustee-agent/README.md

```diff
@@ -0,0 +1,23 @@
+Trustee agent -- a simple tool to attest and fetch secrets from Trustee
+
+Trustee agent is using attestation-agent's kbs_protocol client and
+attesters to gather hardware-based confidential-computing evidence
+and send it over to Trustee.
+
+Trustee agent is a part of [confidential-containers](https://github.com/confidential-containers)
+[guest-components](https://github.com/confidential-containers/guest-components)
+project but can be used for confidential VMs as well.
+
+
+
+Build with:
+    cargo build -p kbs_protocol --bin trustee-agent --no-default-features \\
+          --features "background_check,passport,<openssl|rusttls>,bin,<attesters-list>"
+
+
+Run:
+  $ trustee-agent --url <Trustee-URL> [--cert-file <path>] get-resource --path <resource-path>
```

Comment:
Also this

```suggestion
```bash
./target/debug/trustee-agent --url <Trustee-URL> [--cert-file <path>] get-resource --path <resource-path>
```
```

Reply:
There should also be another ```   the next line. It can not be correctly printed here. THe same to the following

Reply:
I'm not sure what you mean with "another ``` the next line."
I added the ```bash to the patch.

I left it without "./target/debug/" as the program can be also run from e.g. /usr/bin or /usr/local/bin (after cargo install)

Reply:
I added ```   "close" the code-block